### PR TITLE
7.0.4

### PR DESCRIPTION
**Pull request by zsweigart**

### CODE REVIEW

File: ios/RNLinksdk.m-e [hunk 1]
```diff
@@ -53,7 +53,7 @@ NSString* const kRNLinkKitDepositSwitchTokenPrefix = @"deposit-switch-";
 RCT_EXPORT_MODULE();
 
 + (NSString*)sdkVersion {
-    return @"7.0.3"; // SDK_VERSION
+return @"7.0.3"; // SDK_VERSION
```
Comment (pladaub) [inline ios/RNLinksdk.m-e:56]: What is this -e file?

File: ios/RNLinksdk.m [hunk 2]
```diff
@@ -53,7 +53,7 @@ @implementation RNLinksdk
 RCT_EXPORT_MODULE();
 
 + (NSString*)sdkVersion {
-return @"7.0.3"; // SDK_VERSION
+return @"7.0.4"; // SDK_VERSION
```
Comment (pladaub) [inline ios/RNLinksdk.m:56]: nit: indentation

**Review by pladaub — APPROVED** [review]
LGTM after -e file is removed

File: ios/RNLinksdk.m-e (hunk 1 above)
Comment (zsweigart) [reply ios/RNLinksdk.m-e]: not sure, this just gets updated from the version command https://eng-docs.plaid.com/architecture/mobile-sdks/react-native/releasing.html#preparing-a-new-version-commit

**Pull request merged**